### DISCUSSION TITLE
Is there any way to enable vim.o.showmode = true in lazy? When I enable it's automaticlly disabled put I can see the line for a quarter of a second.

### DISCUSSION THREAD

**Discussion by gcdq** [LazyVim/LazyVim#4614] [Q&A] [answered]
title.

**Comment by dotfrag** [comment]
I think `lualine` overrides this, no?

**Reply by gcdq** [reply]
No, I disabled it and it still did not work.

**Reply by dotfrag** [reply]
I'm not entirely sure but I think you also need to set a statusline. It's likely to be empty now.

**Reply by dpetka2001** [reply]
You also have to disable `noice.nvim` (or at least search its docs to see if there's a configurable option). It does something with `showmode` and overwrites it I believe. Also set `cmdheight=1` for `showmode` to work.

**Reply by gcdq** [reply, marked as answer]
If someone still has trouble, I think LazyVim might be overriding the option. I needed to create a config from scratch.
